tests/cli/commands/deposit.test.ts

```diff
@@ -0,0 +1,119 @@
+import {PrivateEth1Network} from "../../../src/eth1/dev";
+import chai, {expect} from 'chai';
+import {DepositCommand} from "../../../src/cli/commands";
+import chaiAsPromised from 'chai-as-promised';
+import logger from "../../../src/logger/winston";
+import {CliError} from "../../../src/cli/error";
+import {Wallet} from "ethers";
+import program from "commander";
+
+chai.use(chaiAsPromised);
+
+describe('[CLI] deposit', () => {
+
+  let eth1Network: PrivateEth1Network;
+
+  before(async () => {
+    logger.silent(true);
+    eth1Network = new PrivateEth1Network({
+      host: '127.0.0.1',
+      port: 32567
+    });
+    await eth1Network.start();
+  });
+
+  after(async () => {
+    await eth1Network.stop();
+    logger.silent(false);
+  });
+
+  it('Should be able to register', async () => {
+    const command = new DepositCommand();
+    const commandCount = program.commands.length;
+    await expect(
+      command.register(program)
+    ).to.not.throw;
+    expect(program.commands.length).to.be.equal(commandCount + 1);
+  });
+
+  it('Should trow error if unable to connect to eth1 network', async () => {
+    const command = new DepositCommand();
+    await expect(
+      command.action(
+        eth1Network.accounts()[0],
+        null,
+        'http://worong_host:123',
+        '32',
+        '0x'
+      )
+    ).to.be.rejectedWith(CliError, 'JSON RPC node (http://worong_host:123) not available.')
+  });
+
+  it('Should trow error if bot private key and mnemonic are not submitted', async () => {
```

Comment:
Can you fix the typos in these test descriptions?

Reply:
yeah sure, I will also add few more tests just to keep everything in line.

Sorry about typos, I'm still adjusting to new keyboard :D

Reply:
congrats on the new keyboard :smiley_cat: